drizzle/0000_tranquil_hardball.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE `job_roles` (
```

Comment:
This and other migration files names should explain their purpose - tranquil hardball doesn't tell us this is actually creating the job roles table :)